src/dotVariant.Generator/Descriptor.cs

```diff
@@ -24,5 +24,12 @@ public sealed record Descriptor(
             var options = Inspect.GetOptions(type);
             return new(type, syntax, options, nullability);
         }
+
+        public string HintName => $"{Type.ToString()
+            // If the contains type parameters replace angle brackets as those are not allowed in AddSource()
```

Comment:
```suggestion
            // If the type contains type parameters replace angle brackets as those are not allowed in AddSource()
```
I know you copied the mistake from my previous code. Would appreciate if we could fix it here.